### ISSUE TITLE
create_form_entry references a payload parameter, but does not seem to send it.

### ISSUE THREAD

**Issue by dcompane** [breid1313/remedy-py#3]
I am looking at how to add attachments to cases on creation, and i noticed the payload parameter is not used, but it is in the definition of a lot of functions.

Any insight as to why?

Any suggestions on how to add the attachments?

(Hope you are still around maintaining this!)

Happy New Year!

**Comment by breid1313** [comment]
Hey Daniel. Thanks for dropping in!

tl;dr -- yes I am still around, but unfortunately I don't have access to Remedy anymore. So it's going to be hard to maintain this going forward.

Regarding the `payload` param, this was a while ago so unfortunately I don't have a definite answer for you. My best guess is that the Remedy API supports some extra set of options that you can add to an incident in some way, but I'm not sure. We may have included that param initially with the intent to follow-up with the code to actually take that extra payload and send it to Remedy.

This is half based off of the code comments and half based on loose memory. Sorry I can't do any better than that at the moment. If the param is confusing I can happily remove it (looks like its actually not used anywhere) along with a couple other little fixes I can see at a glance (ex defaulting kwarg values to `None` instead of instantiating a list/dict.

On attachments... that I have less memory of. I remember talking about adding that down the line once upon a time but it didn't make the cut for this go around. That'd be tough for me to test without a Remedy env.
